Sustainsys.Saml2/WebSSO/MetadataCommand.cs

```diff
@@ -33,14 +33,16 @@ public CommandResult Run(HttpRequestData request, IOptions options)
 
             var urls = new Saml2Urls(request, options);
 
-            var metadata = options.SPOptions.CreateMetadata(urls);
+			IdentityProvider identityProvider = options.Notifications.GetIdentityProvider(options.SPOptions.EntityId, null, options);
```

Comment:
For any customizations on the library, have a descriptive comment about why we need to add this change.

Reply:
added a comment